test/integration/auth.spec.ts

```diff
@@ -135,6 +136,58 @@ describe('admin.auth', () => {
       });
   });
 
+  it('createUser() creates a new user with enrolled second factors', () => {
+    const enrolledFactors = [
+      {
+        phoneNumber: '+16505550001',
+        displayName: 'Work phone number',
+        factorId: 'phone',
+      },
+      {
+        phoneNumber: '+16505550002',
+        displayName: 'Personal phone number',
+        factorId: 'phone',
+      },
+    ];
+    const newUserData: any = {
+      uid: newMultiFactorUserUid,
+      email: generateRandomString(20).toLowerCase() + '@example.com',
+      emailVerified: true,
+      password: 'password',
+      multiFactor: {
+        enrolledFactors,
+      },
+    };
+    return admin.auth().createUser(newUserData)
+      .then((userRecord) => {
+        expect(userRecord.uid).to.equal(newMultiFactorUserUid);
+        // Confirm expected email.
+        expect(userRecord.email).to.equal(newUserData.email);
+        // Confirm second factors added to user.
+        expect(userRecord.multiFactor.enrolledFactors.length).to.equal(2);
+        // Confirm first enrolled second factor.
+        expect(userRecord.multiFactor.enrolledFactors[0].uid).not.to.be.undefined;
```

Comment:
```
const firstFactor = enrolledFactors[0];
```

Reply:
I set `firstMultiFactor` to `userRecord.multiFactor.enrolledFactors[0]`. That improves readability.